cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/group/GcpInstanceGroupResourceBuilder.java

```diff
@@ -45,8 +45,6 @@ public CloudResource build(GcpContext context,
             AuthenticatedContext auth, Group group,
             Network network, Security security, CloudResource resource) throws Exception {
         LOGGER.info("Building GCP instancegroup {} for project {}", group.getName(), context.getProjectId());
-
```

Comment:
Nit: I'm not oposed to adding whitespace to make something more readable, but you only need one line. :)